simulator/state.py

```diff
@@ -206,10 +208,24 @@ def _move_aircraft_laterally(self, time_delta: datetime.timedelta):
     def _move_aircraft_vertically(self, time_delta: datetime.timedelta):
         """
         Evolve the vertical (alt) position of the aircraft.
+
+        Once an altitude greater to or equal to the target altitude is reached, 
+        the rise will adjust to 0.0 in order to maintain target altitude.
         """
 
         dt = time_delta.total_seconds()
+        
+        for i in range((len(self.aircraft.index))):
+            # iterate through each aircraft to change each altitude to the target
+            if self.aircraft["alt"][i]>=self.aircraft["target_alt"][i]:
```

Comment:
If the new target flight level is BELOW the current flight level, does this still work?

Reply:
No. Currently implemented so that if the target flight level is BELOW the current flight level, the rise is adjusted to 0.0 again. Further commits required if we want to evolve the flight so that the flight tries to get to the lower altitude